net/nimble/controller/src/ble_ll_sched.c

```diff
@@ -1078,7 +1078,11 @@ ble_ll_sched_rmv_elem(struct ble_ll_sched_item *sch)
         if (first == sch) {
             os_cputime_timer_stop(&g_ble_ll_sched_timer);
         }
-
+#ifdef BLE_XCVR_RFCLK
+        if (sch == TAILQ_FIRST(&g_ble_ll_sched_q)) {
+            ble_ll_xcvr_rfclk_timer_start(sch->start_time);
+        }
+#endif
```

Comment:
why this is needed now? you will basically restart rfclk for the item you remove in next line so it does not make sense for me. probably what makes sense here is to restart rfclk for 1st element on queue after removing current element but only in case it was the first element (so we basically need to restart rfclk for next element) - but this means we had an issue here and it should be separate patch.

Reply:
this is some terrible copy paste issue - thanks for catching it.